.gitignore

```diff
@@ -1,4 +1,4 @@
-/.php_cs.cache
```

Comment:
not really needed right?

Reply:
now root composer.json is gone we could also drop below right

Reply:
Depends … I was in two minds (this is a common theme for me today) about it, as the people working directly on the repo would want to run PHP-CS-Fixer … so maybe that one should be a leading slash.

Kinda the same with the others, I am not a fan of having `composer.phar` anywhere near a project and think it should be covered in a global gitignore probably the same with the others now I think about it :rofl: